viewer/src/components/ResearchTile.vue

```diff
@@ -50,31 +50,48 @@
 
 <script lang="ts">
 import Vue from "vue";
-import { Component, Prop } from "vue-property-decorator";
-import {
+import {Component, Prop} from "vue-property-decorator";
+import Engine, {
+  AdvTechTilePos,
+  canResearchField,
+  canTakeAdvancedTechTile,
   Condition,
   Event,
   Federation,
   Operator,
   Planet as PlanetEnum,
   Player,
+  PlayerEnum,
   ResearchField,
   researchTracks,
   Resource as ResourceEnum,
   Reward,
 } from "@gaia-project/engine";
-import { descriptions } from "../data/research";
+import {descriptions} from "../data/research";
```

Comment:
The prettier config uses spaces here. It's probably not well configured for .vue files but your IDE shouldn't remove them